### PR TITLE
Content QA for template category

### PR DESCRIPTION
**Pull request by amazingphilippe**
# Summary | Résumé

This PR reviews the translation content and content strings in both languages:

- Updates to FR content in fr.csv
- Changes in templates and py files when English had to be different

# Test instructions | Instructions pour tester la modification

There are changes in a few places. You can do aside-by-side check using [Figma](https://www.figma.com/design/RLcM0JwlQsd2uVK3GJr5qY/Templates-%7C-Gabarits?node-id=3312-1392&t=hO4n2AZc3rySHkwM-4)

Look at these screens in both FR and EN:
1. Platform admin - template categories
2. Platform admin - Single template category
3. Admin - Browse templates (filters expanded)

### CODE REVIEW

File: app/templates/views/templates/_template_list.html [hunk 1]
```diff
@@ -57,7 +60,8 @@
           {% endif %}
         </span>
         <span id="sr-{{ item.id }}1" class="message-meta message-type">
-          {{ _(item.hint) }}
+          {# Compose hint + "template" IF it has a type. ELSE just print the hint #}
+          {{ _("{} template").format(_(item.hint)) | capitalize if item.type else _(item.hint) }}
```
Comment (andrewleith) [inline app/templates/views/templates/_template_list.html:64]: I think this is going to conflict with the work being done in [1910](https://github.com/cds-snc/notification-admin/pull/1910) which was work to address the bugs found in the bug bash. 🤔

Comment (amazingphilippe) [reply app/templates/views/templates/_template_list.html:64]: I think it'll merge fine! 1910 changes the template category part of the template list item. My changes are above these changes:

https://github.com/cds-snc/notification-admin/pull/1910/files#diff-98208c3a75177112fdc7df3059fdf9872e1e87529e9b72eae7a8dd44072647eaL59-L66

Comment (andrewleith) [reply app/templates/views/templates/_template_list.html:64]: My bad, you are right, I got the columns mixed up!

**Comment by YedidaZalik** [comment]
We are waiting for the translation for this figma content:https://www.figma.com/design/RLcM0JwlQsd2uVK3GJr5qY/Templates-%7C-Gabarits?node-id=3330-1476&t=JdTZBNSQkjrRUDWP-0
Marie-Sophie has been tagged.

**Comment by amazingphilippe** [comment]
> We are waiting for the translation for this figma content:https://www.figma.com/design/RLcM0JwlQsd2uVK3GJr5qY/Templates-%7C-Gabarits?node-id=3330-1476&t=JdTZBNSQkjrRUDWP-0
> Marie-Sophie has been tagged.

This would be for #1908 We don't need content for this one!

**Review by jzbahrai — APPROVED** [review]

**Pull request merged**